libs/model/src/models/comment_subscriptions.ts

```diff
@@ -0,0 +1,56 @@
+import { schemas } from '@hicommonwealth/core';
+import type * as Sequelize from 'sequelize'; // must use "* as" to avoid scope errors
+import type { DataTypes } from 'sequelize';
+import { z } from 'zod';
+import type { ModelInstance, ModelStatic } from './types';
+
+export type CommentSubscriptionAttributes = z.infer<
+  typeof schemas.entities.CommentSubscription
+>;
+
+export type CommentSubscriptionInstance =
+  ModelInstance<CommentSubscriptionAttributes>;
+export type CommentSubscriptionModelStatic =
+  ModelStatic<CommentSubscriptionInstance>;
+
+export default (
+  sequelize: Sequelize.Sequelize,
+  dataTypes: typeof DataTypes,
+): CommentSubscriptionModelStatic => {
+  const CommentSubscription = <CommentSubscriptionModelStatic>sequelize.define(
+    'CommentSubscriptions',
+    {
+      id: { type: dataTypes.INTEGER, primaryKey: true, autoIncrement: true },
+      user_id: { type: dataTypes.INTEGER, allowNull: false },
```

Comment:
are you using snake case in subscription schemas for some reason? ... most models use camel case with sequelize underscoring names to make it PG friendly... maybe we need to discuss a convention?

Reply:
Not sure what you are referencing here. All of our model definitions use snake case. All of our Sequelize attribute types and entity schemas use snake case as well. Why would I cause camel case?

Reply:
😱 true... snake case in all schemas it is... need to make sure I'm not using camel case in the new projection.